oidc-conformance-tests/README.md

```diff
@@ -7,10 +7,11 @@ OIDC conformance test workflow can be used to for this purpose.
 2. Click on OIDC conformance test workflow
 3. Click on Run workflow
 4. You need to provide the tag name of the product-is version you want to test (default value is "v5.12.0-m6")
-5. Set send google chat notification to false if you don't want a summary of test results to be sent to a configured google chat group. (default value is true)
-6. Click on Run workflow
-7. After tests are completed you can view test results on the test summary page
-8. Two types of artifacts are saved after the test is completed
+5. Set 'Send test results to email' to 'no' if you don't want to send test summary to a list of pre-configured email addresses. (default is 'yes')
```

Comment:
Can we set the default to no for this option too

Reply:
updated